ideascube/serveradmin/templates/serveradmin/index.html

```diff
@@ -16,11 +16,12 @@
         </ul>
         <hr />
         <ul>
-            <li>{% fa 'cogs' 'fa-fw' %} <a href="{% url 'server:services' %}">{% trans "Manage services" %}</a></li>
-            <li>{% fa 'power-off' 'fa-fw' %} <a href="{% url 'server:power' %}">{% trans "Restart server" %}</a></li>
-            <li>{% fa 'life-ring' 'fa-fw' %} <a href="{% url 'server:backup' %}">{% trans "Manage backups" %}</a></li>
-            <li>{% fa 'plug' 'fa-fw' %} <a href="{% url 'server:battery' %}">{% trans "Monitor battery" %}</a></li>
-            <li>{% fa 'wifi' 'fa-fw' %} <a href="{% url 'server:wifi' %}">{% trans "Manage Wi-Fi" %}</a></li>
+            <li>{% fa 'info-circle' 'fa-fw' %} <a href="{% url 'server:name' %}">{% trans "Server name" %}</a></li>
+            <li>{% fa 'cogs' 'fa-fw' %} <a href="{% url 'server:services' %}">{% trans "Services" %}</a></li>
```

Comment:
Changing labels means that translations need to be updated. We need to be sure we want those change then.